docs/users.md

```diff
@@ -237,7 +237,11 @@ Then, you can define quotas on this set:
 - max disk usage
 
 :::tip
-Snapshotting a VM within a self-service will _not_ use the quota from the resource set. The same rule applies for backups and replication.
+Replicated VMs and snapshots created by a backup job don't use quotas.
+:::
+
+:::tip
+A snapshot of a Self Service VM will use as much resources as a VM would. You can disable this by setting `enableSnapshotConsumption` to `false` in the `selfService` section of `xo-server`'s config.
```

Comment:
Wouldn't it be better to have a `ignoreVmSnapshotResources` to set to `true` instead?

Reply:
I tried to avoid stacking negations but I can change it.

Reply:
Setting *ignore* to *true* is not a double negation.

But I find it clearer to do this to disable a feature instead of setting a value to `false`.

Reply:
I meant in the code, if you want to know if you have to take the snapshots into account, you'll have a condition like `!ignore...`, which is a double negation.

> But I find it clearer to do this to disable a feature instead of setting a value to `false`.

Ok, I'll change it then :+1: